examples/vue3-ts-todo-list/src/App.vue

```diff
@@ -27,18 +33,27 @@
 
 <script>
 import Auth from '@/components/Auth.vue'
+import PasswordReset from '@/components/PasswordReset.vue'
 import TodoList from '@/components/TodoList.vue'
 import Loading from '@/components/Loading.vue'
 import Footer from '@/components/Footer.vue'
 import { userSession, handleLogout } from '@/vuetils/useAuth'
+import { getParameterByName } from './lib/helpers'
```

Comment:
To make it more consistent with the other import statements.

```suggestion
import { getParameterByName } from '@/lib/helpers'
```